.github/workflows/publish.yml

```diff
@@ -82,3 +82,15 @@ jobs:
         run: npx vsce publish --no-git-tag-version --packagePath $(find . -iname *.vsix)
         env:
           VSCE_PAT: ${{ secrets.VSCE_PAT }}
+
+  publish-ovsx:
+    name: Publish All (OpenVSX)
+    runs-on: ubuntu-latest
+    needs: build
+    if: success() && startsWith( github.ref, 'refs/tags/v')
+    steps:
+      - uses: actions/download-artifact@v3
+      - name: Publish Extension
+        run: npx ovsx publish --packagePath $(find . -iname *.vsix)
```

Comment:
```suggestion
        run: npx ovsx publish --packagePath $(find . -iname '*.vsix' ! -name '*web*')
```

We now publish a web version (https://github.com/hashicorp/vscode-terraform/pull/1210) of the extension as well, which — _I think_ — we need to filter here. This change should ensure we only upload the platform-specific packages while skipping the web one.

Reply:
Happy to commit this, @dbanck, and thanks for taking care of the publishing. 

I'm unsure if this is a necessary change, though: if the extension is offered in `web` form as well, it will work properly for VS Code Web, otherwise, it shouldn't be labelled as compatible.

From the VS Code Docs: [[1](https://code.visualstudio.com/api/working-with-extensions/publishing-extension#platformspecific-extensions)]

> If you want a platform-specific extension to also support running in the browser as a [web extension](https://code.visualstudio.com/api/extension-guides/web-extensions), it must target the web platform when publishing. The web platform will respect the browser entry point in the package.json. The extension capabilites that are not supported in the web should use when clauses in the package.json to make sure that they are disabled. We do not recommend that extensions ship a different package.json for the web platform or to remove parts of the VSIX that do not work in the web.

Reply:
We offer a web version of the extension, which works in VS Code Web. We added an additional target to our publishing pipeline and skipped bundling the language server (https://github.com/hashicorp/vscode-terraform/pull/1262).

We weren't sure if OpenVSX also supports `web` as a target when publishing, so I proposed filtering the package. Fantastic if that's not required and there is `web` target support! Do you know if that's the case? I did not try to publish the web `vsix` because I didn't want to break anything.

Reply:
You were right, this change isn't necessary, and OpenVSX supports `web` targets.

<img width="328" alt="CleanShot 2022-11-23 at 19 55 19@2x" src="https://user-images.githubusercontent.com/45985/203734024-4a1cd619-c702-492d-837b-ea0664a37775.png">